README.md

```diff
@@ -27,6 +27,23 @@ $> docker run --name=ntp             \
               cturra/ntp
 ```
 
+Load your own NTP config file into the container
+---
+
+Add 
+
+```-v /path/to/folder/containing/my_config_file:/data```
```

Comment:
generally, unless absolutely necessary, i prefer not to mount an external volume. in this case, i'm not convinced it's needed, per my comment above about including a custom `assets/ntpd.conf` file.